app/controllers/vacancies_controller.rb

```diff
@@ -26,6 +26,8 @@ def index
 
     @vacancies = VacanciesPresenter.new(records, searched: @filters.any?)
 
+    audit_search_event if @filters.any?
```

Comment:
Surprised to see we are adding this line in in this commit, is this right?

Reply:
Yeah, I agonised over this line a bit, as I initially put in in the finer, but I think it's OK here. I might actually stick it in the service object now